pages/apim/3.x/user-guide/publisher/expression-language.adoc

```diff
@@ -312,9 +312,14 @@ is based on an API property named `my-property`:
 
 == Conditions
 
-Expression language can also be used to set condition of execution [see conditional policies & flows conditions] and it is possible use logical operator as `&&` or `||` :
+You can also use the Expression Language to set a condition of execution (see 'conditional policies and flows conditions') and it is possible to use logical operators such as `&&` or `||`, as shown in the example below:
+
+`{#request.headers['my-header'] != null && #request.headers['my-header'][0] == "my-value"}`
+
+WARNING: Using this construct in a mock policy where the header is not available returns a `500` error because of a header `null` value.
```

Comment:
You can remove that. it was just an indication for you when using `request.headers.getFirstValue("my-header")` which can lead to this error.
But do not add it to the documentation

```suggestion
WARNING: Using this construct in a mock policy where the header is not available returns a `500` error because of a header `null` value.
```